src/status_im2/subs/onboarding.cljs

```diff
@@ -69,9 +69,9 @@
 (defn login-ma-keycard-pairing
```

Comment:
someoutdated info here, should be ->
```
(defn login-profile-keycard-pairing
  "Compute the keycard-pairing value of the profile selected for login"
  ```
  ???

Reply:
yeah i did these changes with the refactoring tool, so there are more places which will be addressed later, but now you pointed it i'll fix it in this PR

Reply:
cool, yeah of course. What's the refactoring tool? 🤔

Reply:
in the Intellij IDEA we have a refactoring tool, so it renames all usages of keyword or symbol when you rename it

Reply:
Oh nice 👌